README.md

```diff
@@ -100,21 +99,23 @@ cd ..
 ```
 
 Train BigVGAN model. Below is an example command for training BigVGAN-v2 using LibriTTS dataset at 24kHz with a full 100-band mel spectrogram as input:
+
 ```shell
 python train.py \
 --config configs/bigvgan_v2_24khz_100band_256x.json \
---input_wavs_dir LibriTTS \
---input_training_file LibriTTS/train-full.txt \
---input_validation_file LibriTTS/val-full.txt \
---list_input_unseen_wavs_dir LibriTTS LibriTTS \
---list_input_unseen_validation_file LibriTTS/dev-clean.txt LibriTTS/dev-other.txt \
+--input_wavs_dir tests/LibriTTS \
```

Comment:
I think for the bundled LibriTTS filelists, locating them into `filelists` may be better than putting them into `tests`.